tests/data_test.py

```diff
@@ -266,7 +266,7 @@ def test_normalize(tmp_path: Path):
     data.normalizations = data.get_normalizations()
     test_data = data.normalize(data.observed)
     assert np.all(
-        np.hstack(data.normalizations[None].values()).tolist()
+        np.hstack(list(data.normalizations[None].values())).tolist()
```

Comment:
```suggestion
        np.hstack(list(data.normalizations[None])).tolist()
```

Reply:
Hmm not sure? `data.normalizations[None]` would technically be the keys, not the values?

Reply:
oh, right.. sorry!